sound/soc/sof/Makefile

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
 
 snd-sof-objs := core.o ops.o loader.o ipc.o pcm.o pm.o debug.o topology.o\
-		control.o trace.o utils.o sof-audio.o stream-ipc.o ipc4.o
+		control.o trace.o iomem-utils.o sof-audio.o stream-ipc.o ipc4.o
```

Comment:
typo in commit message: The latter is used by the PCM

Reply:
> > Looks like we have Intel CI issues with configurations in https://sof-ci.01.org/linuxpr/PR3136/build6666/devicetest/, redoing the tests.
> 
> https://sof-ci.01.org/linuxpr/PR3136/build6668/devicetest/ shows almost the same pattern of failures. That pattern is new and hasn't been seen anywhere else so I know it's always tempting but maybe not blame the messenger this time? ([thesofproject/sof-test#802](https://github.com/thesofproject/sof-test/pull/802)). Maybe we've finally reached a point where most PR failures are actually caused by the code change itself? :-)

This PR is one of the most intrusive one we have had in recent history, I would be extra paranoid when looking at the results.
 
> Connecting some dots, my vaguely educated guess is that this PR breaks on one particular platform the `Copying cached debugfs data` feature.

This is coming from the core itself (debug.c) which has not been touched and it is printed because the device is suspended so we are returning the content of the cached buffer.